src/styles/index.scss

```diff
@@ -1,3 +1,132 @@
+$main-f-color: #060b35;
+$secondary-f-color: #616070;
+$blue-accent: #00acdc;
```

Comment:
you can create a separate `variables.scss` file and store your colors there. (just import this file into `index.scss` using `@import`)